clusterloader2/api/validation.go

```diff
@@ -18,24 +18,186 @@ package api
 
 import (
 	"fmt"
+	"os"
+
+	"k8s.io/apimachinery/pkg/util/validation"
+	"k8s.io/apimachinery/pkg/util/validation/field"
 	"k8s.io/perf-tests/clusterloader2/pkg/errors"
 )
 
+// ConfigValidator contains metadata for config validation
```

Comment:
NIT: It's recommended that the doc comments are complete sentences, i.e. they should end with the dot (.) - https://golang.org/doc/effective_go.html#commentary